uc.c

```diff
@@ -211,7 +223,7 @@ static gint uc_exits_cmp(gconstpointer a, gconstpointer b, gpointer user_data)
     }
 }
 
-static uc_err uc_init(uc_engine *uc)
+static uc_err uc_init_engine(uc_engine *uc)
```

Comment:
Any reason for these renames?

Reply:
I changed `<arch>_uc_init` to `uc_init_<arch>` via symbols.sh in d68013d7745fd2de0628de721cbe250df36619c9, mainly to make things a bit more uniform across architectures. Since that #define's uc_init, I had to change the name here to reduce confusion. It should not be a big issue given that these are static.